packages/deploy/deploy/400_asset/402_deploy_asset.ts

```diff
@@ -43,4 +43,8 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 export default func;
 
 func.tags = ['Asset', 'Asset_deploy', 'L2'];
-func.dependencies = ['TRUSTED_FORWARDER_V2', 'OperatorFilter_setup'];
+func.dependencies = [
+  'TRUSTED_FORWARDER_V2',
+  'RoyaltyManager',
```

Comment:
I think this should be `RoyaltyManager_deploy`
(TRUSTED_FORWARDER_V2 is external, hence it doesn't have the _deploy part)

Reply:
Updated